sql/cagg_migrate.sql

```diff
@@ -510,6 +518,12 @@ BEGIN
         WHERE mat_hypertable_id OPERATOR(pg_catalog.=) _plan_step.mat_hypertable_id
         AND step_id OPERATOR(pg_catalog.=) _plan_step.step_id;
         COMMIT;
+
+        -- SET LOCAL is only active until end of transaction.
+        -- While we could use SET at the start of the function we do not
+        -- want to bleed out search_path to caller, so we do SET LOCAL
+        -- again after COMMIT
+        SET LOCAL search_path TO pg_catalog, pg_temp;
```

Comment:
Doesn't this mean that the search path is set for the transaction following the loop?